torch/_functorch/config.py

```diff
@@ -26,6 +27,26 @@
 
 debug_partitioner = os.environ.get("AOT_PARTITIONER_DEBUG", False)
 
+# Today, if you are in a situation where there is "false aliasing"
```

Comment:
@ezyang my attempt at using JK is here, with the corresponding knob [here](https://www.internalfb.com/intern/justknobs/?name=pytorch%2Fdynamo) (I threw it under dynamo configs because a dedicated namespace felt like overkill.

I was also paranoid about the fact that `getval_int()` defaults to zero if JK isn't available for some reason. LMK if this looks reasonable to you (I cargo-culted this a bit from inductor's JK usage)

Reply:
Meh, don't do it this way. Do a boolean JK and just have it be a killswitch. (When JK is not available, the killswitch should default to "not killed", because the steadystate is for the killswitch to be off.)